src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -247,6 +256,15 @@ public KeyspaceMetrics(final Keyspace ks)
 
         clientReadSizeWarnings = createKeyspaceMeter("ClientReadSizeWarnings");
         clientReadSizeAborts = createKeyspaceMeter("ClientReadSizeAborts");
+        clientReadSize = createKeyspaceHistogram("ClientReadSize", false);
+
+        clientLocalReadSizeTooLargeWarnings = createKeyspaceMeter("ClientLocalReadSizeTooLargeWarnings");
+        clientLocalReadSizeTooLargeAborts = createKeyspaceMeter("ClientLocalReadSizeTooLargeAborts");
+        clientLocalReadSize = createKeyspaceHistogram("ClientLocalReadSize", false);
```

Comment:
The internals we can rename any time, but in terms of the externally facing metric names, I'd consider something like `CoordinatorReadSize`/`LocalReadSize` for the histograms. (The token "Client" seems to be there mostly to indicate "things we send to the client", and these histograms could exist independently.)

Reply:
Also a note to self: Check to see that the histograms are updated even if warnings/aborts aren't enabled...

Reply:
Also, same comments apply to `TableMetrics`, of course.

Reply:
so much pain but updated all places to have consistent names -_-

best to do before it goes live =)